ui/ducks/send/send.js

```diff
@@ -636,25 +626,6 @@ export const initializeSendState = createAsyncThunk(
       metamask: { gasFeeEstimates, gasEstimateType },
     } = thunkApi.getState();
 
-    if (sendState.stage !== SEND_STAGES.EDIT) {
-      // Because we are only interested in getting a gasLimit estimation we only
-      // need to worry about gasPrice. So we use maxFeePerGas as gasPrice if we
-      // have a fee market estimation.
-      if (gasEstimateType === GasEstimateTypes.legacy) {
-        gasPrice = getGasPriceInHexWei(gasFeeEstimates.medium);
-      } else if (gasEstimateType === GasEstimateTypes.ethGasPrice) {
-        gasPrice = getRoundedGasPrice(gasFeeEstimates.gasPrice);
-      } else if (gasEstimateType === GasEstimateTypes.feeMarket) {
-        gasPrice = getGasPriceInHexWei(
-          gasFeeEstimates.medium.suggestedMaxFeePerGas,
-        );
-      } else {
-        gasPrice = gasFeeEstimates.gasPrice
-          ? getRoundedGasPrice(gasFeeEstimates.gasPrice)
-          : '0x0';
-      }
-    }
```

Comment:
I don't think gasPrice is necessary since we ultimately ask provider to get `gasLimit`